src/tcgwars/logic/impl/gen4/Unleashed.groovy

```diff
@@ -1143,7 +1143,9 @@ public enum Unleashed implements LogicCardInfo {
             energyCost W
             onAttack {
               damage 10
-              damage 10, opp.bench.select()
+              if opp.bench {
+                damage 10, opp.bench.select()
```

Comment:
It's also helpful to add a `string` argument in `select()` to help the user what they're selecting and why